src/context/i18n/index.js

```diff
@@ -0,0 +1,60 @@
+import React, { createContext, useContext, useEffect, useState } from "react";
+import { mapTranslationsToFormioI18nObject } from "./translationsMapper";
+
+export const supportedLanguages = ["nb-NO", "nn-NO", "en", "pl"];
+export const languagesInNorwegian = {
+  "nb-NO": "Norsk bokmål",
+  "nn-NO": "Norsk nynorsk",
+  en: "Engelsk",
+  pl: "Polsk",
+};
+export const languagesInOriginalLanguage = {
+  "nb-NO": "Norsk bokmål",
+  "nn-NO": "Norsk nynorsk",
+  en: "English",
+  pl: "Polskie",
+};
+
+const I18nContext = createContext({});
+
+function I18nProvider({ children, loadTranslations }) {
+  const [translations, setTranslations] = useState({});
+  const [translationsForNavForm, setTranslationsForNavForm] = useState(null);
+  const [currentTranslation, setCurrentTranslation] = useState({});
+
+  useEffect(() => {
```

Comment:
May cause redudant loading of translations. Check if we can remove/move this logic.